pkg/v1/sdk/capabilities/discovery/tkg/capabilities.go

```diff
@@ -36,9 +37,11 @@ func (dc *DiscoveryClient) IsTKGm(ctx context.Context) (bool, error) {
 	return dc.HasInfrastructureProvider(ctx, InfrastructureProviderVsphere)
 }
 
-// IsTKGS returns true if the cluster is a TKGS cluster.
+// IsTKGS returns true if the cluster is a TKGS cluster. Check for TKGS on any TKC API version */
```

Comment:
```suggestion
// IsTKGS returns true if the cluster is a TKGS cluster. Checks for the existence of any TKC API version
```

Reply:
Corrected.

Reply:
I don't think the suggested change was committed?